src/manifolds/VectorBundle.jl

```diff
@@ -287,18 +283,6 @@ function get_coordinates(M::VectorBundle, p, X, B::ArbitraryOrthonormalBasis) wh
     coord2 = get_coordinates(M.fiber, px, VXF, B)
     return vcat(coord1, coord2)
 end
-function get_coordinates(
```

Comment:
Why did you remove this function?

Reply:
All that were previously `PrecomputedXBasis` based functions are now handled by the single `CachedBasis` function, which should be the one l. 168 or 171 in `bases.jl`. If I understood all implementations correctly (and the tests are still running, but I still have to check for coverage).

Reply:
These two methods in `bases.jl` are just generic implementations. We should still have specialized ones for product, power and vector bundle (they are much faster).